example/types/ExampleContract.py

```diff
@@ -23,10 +23,17 @@
 # consumers have too many opinions on line length
 # pylint: disable=line-too-long
 
+# We use protected classes within pypechain
```

Comment:
idc about this disable, but is it better to make the classes we use not protected?